Makefile

```diff
@@ -65,3 +65,105 @@ build-image:
 .PHONY: push-image
 push-image:
 	docker push $(IMAGE)
+
+.PHONY: clean
+clean:
+	rm -rf ./build
+
+.PHONY: sdk-build
+sdk-build:
+	npm ci
+	npm run build
+
+.PHONY: react-native-npm-ci
+react-native-npm-ci:
+	cd ./example/reactnative; \
+		rm -rf node_modules; \
+		yarn install --frozen-lockfile
+
+.PHONY: react-native-set-versionCode
+react-native-set-versionCode:
+	/usr/bin/sed -I "" "s/versionCode 1/versionCode $(shell date +%s)/" ./example/reactnative/android/app/build.gradle
+
+.PHONY:	react-native-build-unsigned-apk
+react-native-build-unsigned-apk:
+	cd ./example/reactnative/android; \
+		./gradlew :app:assembleRelease
+
+.PHONY: react-native-zipalign
+react-native-zipalign:
+	"$(ANDROID_HOME)/build-tools/33.0.3/zipalign" -c -v 4 ./example/reactnative/android/app/build/outputs/apk/release/app-release-unsigned.apk
+
+.PHONY: react-native-apksigner
+react-native-apksigner:
+	"$(ANDROID_HOME)/build-tools/33.0.3/apksigner" sign \
+		--ks $(ANDROID_KEYSTORE_PATH) \
+		--ks-key-alias $(ANDROID_KEY_ALIAS) \
+		--ks-pass pass:$(ANDROID_KEYSTORE_PASSWORD) \
+		--key-pass pass:$(ANDROID_KEY_PASSWORD) \
+		--out ./example/reactnative/android/app/build/outputs/apk/release/app-release-signed.apk \
+		./example/reactnative/android/app/build/outputs/apk/release/app-release-unsigned.apk
```

Comment:
1. I believe you should be able to handle all this with [fastlane](https://docs.fastlane.tools/actions/build_android_app/)

2. I think playstore requires an aab instead of apk.

Reply:
I just moved the code from the YAML file to here. Will use fastlane to distribute in another PR.